projects/solidity/build.sh

```diff
@@ -27,12 +27,13 @@ cd $SRC/boost
 
 # Build solidity
 cd $SRC/solidity
-git submodule update --init --recursive
```

Comment:
Hm, we do not have submodules, but having `--recursive` will not hurt.

Reply:
okay, the `--recursive` argument has been moved to the dockerfile now.